wormhole-connect/src/views/v2/Redeem/index.tsx

```diff
@@ -276,19 +292,101 @@ const Redeem = () => {
     let etaElement: string | ReactNode = <CircularProgress size={14} />;
 
     if (eta) {
-      etaElement = millisToHumanString(eta);
+      etaElement = millisToMinutesAndSeconds(eta);
     }
 
     return (
       <Stack alignItems="center" justifyContent="center">
         <Typography color={theme.palette.text.secondary} fontSize={14}>
           ETA {etaElement}
         </Typography>
-        <Typography fontSize={24}>{counter}</Typography>
+        <Typography fontSize={28}>{counter}</Typography>
       </Stack>
     );
   }, [eta, etaExpired, isRunning, minutes, seconds]);
 
+  // Value for determinate circular progress bar
+  const etaProgressValue = useMemo(() => {
+    const timePassed = Date.now() - txTimestamp;
+
+    // Check if eta has already
+    if (timePassed >= eta) {
+      return 100;
+    }
+
+    return (timePassed / eta) * 100;
+  }, [eta, txTimestamp, seconds]);
+
+  // In-progress circular progress bar
+  const etaCircularProgress = useMemo(() => {
```

Comment:
I wasn't sure but can move this progress bar in its own file. It's not used anywhere else.